module/VuFind/src/VuFind/Controller/InstallController.php

```diff
@@ -821,7 +820,22 @@ public function performsecurityfixAction()
                     $row->save();
                 }
             }
-            $msg = count($rows) . ' user row(s) encrypted.';
+            $msg = count($userRows) . ' user row(s) encrypted.';
+            $this->flashMessenger()->addMessage($msg, 'info');
+        }
+        $cardRows = $this->getTable('usercard')->getInsecureRows();
+        if (count($cardRows) > 0) {
+            // Create a dummy user for encryption purposes...
+            $dummyUser = $this->getTable('user')->createRow();
+            foreach ($cardRows as $row) {
+                if ($row->cat_password) {
```

Comment:
Since cat_password can't be null here, is this check appropriate or should an empty string be encrypted anyway? Otherwise we'll end up not converting all "insecure" cards, right?

Reply:
You're right -- the check was carried forward when I copied and pasted the user row logic (which had to check multiple columns for possible repairs), but it doesn't apply in this context since there's only one column under consideration, so I have removed it.